Aeickelman40/Whats-Cookin-Refactor-Tractor
# Feature/shopping list button

#### What's this PR do?
Adds a view shopping list button to the dom in order to display the items that the user does not have in their pantry that is required to cook a recipe. Also adds ingredient amounts and units to the view pantry button. Ingredient amount for the Shopping list also needs fixing.

ALSO

Commented out all files within the Data folder as we are now pulling from the API, do not panic when you see the changed line count :) 
#### Where should the reviewer start?
scripts.js - line 222 and 257
#### How should this be manually tested?
Select the 'View Pantry' and 'View Shopping List' buttons in the upper navigation bar. Note you need to add recipes first in order to see the ingredients within the shopping list.
